@FoamyGuy
The builds are failing with
```
Error: reuse....................................................................Failed
- hook id: reuse
- exit code: 1

Traceback (most recent call last):
  File "/home/runner/.cache/pre-commit/repo_bey_7z_/py_env-python3/bin/reuse", line 3, in <module>
    from reuse._main import main
  File "/home/runner/.cache/pre-commit/repo_bey_7z_/py_env-python3/lib/python3.14/site-packages/reuse/__init__.py", line 15, in <module>
    from pkg_resources import DistributionNotFound, get_distribution
ModuleNotFoundError: No module named 'pkg_resources'
```
Does this ring a bell with you? There's a lot that's out of date in this Blinka library: the pre-commit tool versions are old, it's still using pylint, etc., but I'm not sure what is causing this particular problem. Do you have library patches that would bring this up to date automatically? Thanks.

--- a/.pre-commit-config.yaml
+++ b/.pre-commit-config.yaml
@@ -8,7 +8,7 @@ repos:
     hooks:
       - id: black
   - repo: https://github.com/fsfe/reuse-tool
-    rev: v1.1.2
+    rev: v6.2.0
     hooks:
       - id: reuse
   - repo: https://github.com/pre-commit/pre-commit-hooks

--- a/docs/conf.py
+++ b/docs/conf.py
@@ -94,17 +94,7 @@ napoleon_numpy_docstring = False
 #
 on_rtd = os.environ.get("READTHEDOCS", None) == "True"
 
-if not on_rtd:  # only import and set the theme if we're building docs locally
-    try:
-        import sphinx_rtd_theme
-
-        html_theme = "sphinx_rtd_theme"
-        html_theme_path = [sphinx_rtd_theme.get_html_theme_path(), "."]
-    except:
-        html_theme = "default"
-        html_theme_path = ["."]
-else:
-    html_theme_path = ["."]
+html_theme = "sphinx_rtd_theme"
 
 # Add any paths that contain custom static files (such as style sheets) here,
 # relative to this directory. They are copied after the builtin static files,

